docs/agents/PROJECT_OBJECTIVES.md

```
@@ -188,7 +220,7 @@ Next improvements:
  - No model binaries in git or Git LFS.
  - No replacement of Retrosheet raw data with transformed-only tables.
 
- ## New Additions (April 2026)
+ ## New Additions (April‑May 2026)
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)